universal-application-tool-0.0.1/conf/evolutions/default/3.sql

```diff
@@ -9,7 +9,12 @@ create table if not exists applicants (
 
 create table if not exists questions (
   id bigserial primary key,
-  object jsonb not null
+  version bigserial,
+  path varchar,
+  name varchar,
+  description varchar,
+  question_text jsonb,
+  question_help_text jsonb
```

Comment:
question (non-blocking): what's the plan for persisting the question's question type?

Reply:
We're working on it - we need to persist the QuestionType enum safely, then reform the question into the proper class on load. We're working on this today (Friday)